tests/test_ingest.py

```diff
@@ -69,7 +76,7 @@ def test_find_root_directory(pipeline, sessions_csv):
     root_dir = find_root_directory(ephys_root_data_dir, session_full_path)
 
     assert root_dir.as_posix() == get_ephys_root_data_dir()
```

Comment:
```suggestion
    session_full_path = pathlib.Path('/main/workflow-array-ephys/tests/user_data') / sess.session_dir
    root_dir = find_root_directory(ephys_root_data_dir, session_full_path)

    assert root_dir.as_posix() == '/main/workflow-array-ephys/tests/user_data'
```

Reply:
Add remove the previous line
```
session_full_path = pathlib.Path(get_ephys_root_data_dir()) / sess.session_dir
```

Reply:
Modifying this test should still allow for testing the `find_root_directory` function.  Do you think this works correctly?

Reply:
That makes sense - I think it should work. We would still test that the data was downloaded with the relative directories as intended. I'll run the tests to be sure.